site/en/docs/devtools/css/reference/index.md

```diff
@@ -436,20 +436,35 @@ Here's a description of each of the UI elements of the **Color Picker**:
     or a page colors palette. DevTools generates the page color palette based on the colors that it
     finds in your stylesheets.
 
-#### Sample a color off the page with the Eyedropper {: #eyedropper }
+#### Sample a color (anywhere) with the Eyedropper {: #eyedropper }
 
 When you open the **Color Picker**, the **Eyedropper**
-{% Img src="image/admin/FCjp9jpqJo8tAB7LneFU.png", alt="Eyedropper", width="28", height="28" %} is on by default. To change
-the selected color to some other color on the page:
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/WKeaXT922ot9wQjtvwcZ.svg", alt="Eyedropper.", width="20", height="20" %} is on by default.
 
-1.  Hover over the target color in the viewport.
-2.  Click to confirm.
+The **Eyedropper** {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/WKeaXT922ot9wQjtvwcZ.svg", alt="Eyedropper", width="20", height="20" %} can sample colors both from the page and, with a corresponding experiment enabled, from anywhere on the screen:
```

Comment:
Don't need to mention the experiment anymore. It consider lands. Usually the team will leave the experiment and enable by default for a couple of cycles in case of any unforeseen issues. The code clean up will happen after.

Anyway, it's consider land.